website/src/components/SideMenuLayout.tsx

```diff
@@ -13,10 +13,16 @@ export const SideMenuLayout = (props: SideMenuLayoutProps) => {
   return (
     <Box backgroundColor={colorMode === "light" ? "gray.100" : colors.dark.bg} className="sm:overflow-hidden">
       <Box display="flex" flexDirection={["column", "row"]} h="full" gap={["0", "0", "0", "6"]}>
-        <Box p={["3", "3", "3", "6"]} pr={["3", "3", "3", "0"]}>
+        <Box p={["3", "3", "3", "6"]} pe={["3", "3", "3", "0"]}>
           <SideMenu buttonOptions={props.menuButtonOptions} />
         </Box>
-        <Box className="p-3 lg:p-6 lg:pl-1 w-full">{props.children}</Box>
+        <Box
+          className="p-3 lg:p-6 ltr:lg:pl-0 rtl:lg:pr-0 w-full"
+          overflowX="hidden"
+          overflowY={{ base: "hidden", sm: "auto" }}
```

Comment:
it is only hidden on `xs`? it feels to me that you can remove the `base` item, or maybe remove the overflowY entirely?

Reply:
It's hidden in xs to avoid the double scroll bar which is really annoying and auto on sm to allow scrolling